DBMS/CommandJoin/question1/Create_Script.sql

```diff
@@ -0,0 +1,68 @@
+drop database StoreFront;
+create database StoreFront;
+use StoreFront;
+
+create table User (User_ID varchar(25) NOT NULL PRIMARY KEY, Name varchar(25));
+
+create table Shopper(Shopper_ID varchar(25),FOREIGN KEY (Shopper_ID) 
+                     references User(User_ID), Name varchar(25));
```

Comment:
There should always be a primary key in each table. It is missing in this table.